src/Materials/Node/Blocks/PBR/subSurfaceBlock.ts

```diff
@@ -146,7 +146,7 @@ export class SubSurfaceBlock extends NodeMaterialBlock {
 
         #ifdef SUBSURFACE
             vec2 vThicknessParam = vec2(0., ${thickness});
-            vec4 vTintColor = vec4(${tintColor}, ${refractionTintAtDistance});
+            vec4 vTintColor = vec4(toLinearSpace(${tintColor}), ${refractionTintAtDistance});

```

Comment:
The tint colour needs to be linearized but my change here doesn't seem to accomplish that. @sebavan am I doing something silly here?

Reply:
The `tintColor` must not be linearized here: it's the responsibility of the user to linearize it in the NME (there is a switch on the texture block for that, as well as a switch on the color3/color4 block, so it's easy to do).

Reply:
Also, the NME sub surface block will need to be updated to handle the new `surfaceAlbedo` parameter passed to the `subSurfaceBlock(...)` shader call.

Also, (optionally?) the `SS_ALBEDOFORTRANSLUCENCYTINT` could be handled as a new switch on the SubSurface block.

Reply:
Thanks, @Popov72 .